src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -36,8 +36,7 @@ const DiscoverySearch = ({
   fetchDiscoveryData = () => {},
   fetchData,
   onCancelToken = () => {},
-  hideBackIcon = false,
-  autoFocus = true
+  hideBackIcon = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `autoFocus` prop might affect the user experience as the search bar will not be focused automatically when the component is rendered. However, this change seems to be compensated by the addition of a `useEffect` hook that sets focus on the search bar after a delay.

```diff
-  autoFocus = true
+  hideBackIcon = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->